### DISCUSSION TITLE
Paypal support

### DISCUSSION THREAD

**Discussion by johan-naizu** [community/community#4185] [Other Feature Feedback, Questions, & Ideas] [answered]
Does GitHub sponsorship support paypal

**Comment by danielcompton** [comment, marked as answer]
Hi!

GitHub Sponsors supports PayPal, Credit Card, and invoiced billing as payment methods. For payouts to maintainers, we currently only offer Stripe Connect.

Does that answer your question? What were you wanting to do with PayPal?

**Reply by MidSpike** [reply]
> Starting on February 23, 2023, GitHub Sponsors will no longer support PayPal as a payments processor. Sponsors that currently use PayPal will be required to update their payment method to pay by credit or debit card.

**Reply by stephanecharette** [reply]
Also see https://github.com/orgs/community/discussions/45180 about the recent termination of support for PayPal.
